docs/index.rst

```diff
@@ -449,6 +449,78 @@ following flag or environment variable::
 
     $ PYTHONWARNINGS=all python app.py
 
+.. _opentracing:
+
+OpenTracing
+-----------
+
+Datadog OpenTracing Tracer
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Datadog opentracer provides an OpenTracing-compatible API to the Datadog
+tracer so that you can use the Datadog tracer in your OpenTracing-compatible
+applications.
+
+Installation
+~~~~~~~~~~~~
+
+To include OpenTracing dependencies in your project with `ddtrace`, ensure you
+have the following in `setup.py`::
+
+    install_requires=[
+        "ddtrace[opentracing]",
+    ],
+
+Configuration
+~~~~~~~~~~~~~
+
+The OpenTracing convention for initializing a tracer is to define an
+initiliazation method that will configure and instantiate a new tracer and
+overwrite the global `opentracing.tracer` reference.
+
+Typically this method looks something like::
+
+    from opentracing.scope_managers import ThreadLocalScopeManager
+    from ddtracer.opentracer import Tracer
+
+    def init_tracer(service_name):
+        config = {
+          'agent_hostname': 'localhost',
+          'agent_port': 8126,
+          'debug': False,
+          'enabled': True,
+          'global_tags': {},
```

Comment:
keep only:
```python
config = {
    'agent_hostname': 'localhost',
    'agent_port': 8126,
}
```
because are the most common configs developers are changing.

Also `scope_manager=ThreadLocalScopeManager` is not mandatory so let's remove it. I'd say to describe the interface, so that people knows how to set the `scope_manager`, plus link to OT docs / repo for available scope managers. I prefer a snippet that is a kind of "copy-paste ready" for common cases. Also I've noticed that [here](https://github.com/DataDog/dd-trace-py/blob/e3f2ac4076fed1a422f20bb4cfac51e51b4ca617/ddtrace/opentracer/tracer.py#L60) we expect the instance and not the class so in any case it should be `ThreadLocalScopeManager()` right?